Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -187,11 +190,15 @@ def bottle_formula(f)
       rebuild = rebuilds.empty? ? 0 : rebuilds.max.to_i + 1
     end
 
-    filename = Bottle::Filename.create(f, Utils::Bottles.tag, rebuild)
+    compression_type = :xz
+    compression_type = :gzip if ARGV.include?("--use-gzip") || f.name == "xz"
```

Comment:
I don't know if it's worth being explicit about this? Such as throwing an `ohai "Using gzip rather than xz to bottle xz"` or something. This isn't really public-facing code in the sense that users don't routinely use it locally, but it feels a bit "eh" to make folks read the code to understand what's going on.

Reply:
@DomT4 The output spits out the filename already:
`Bottling wget-1.19.2_1.high_sierra.bottle.tar.gz...`

Reply:
@MikeMcQuaid Aye, I know. Just wondering if that's explicit enough or whether the concept that `xz` shouldn't be bottled using `xz` needs some kind of brief further elaboration.